support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -63,6 +45,10 @@ function Heading({
 
 	const contributorName = name && name.length < 10 ? name : '';
 
+	if (!amount) {
```

Comment:
Why not reverse this and put `if (amount)`? Then only need the single dropout ` <GenericHeading` at the bottom?

Reply:
I quite like the explicitness of this, it makes it really clear how we're handing the case where we don't have an amount 👍